src/Sfa.Tl.Matching.Data/Repositories/MatchingServiceOpportunityReport.sql

```diff
@@ -0,0 +1,177 @@
+IF OBJECT_ID('tempdb..#Roles') IS NOT NULL 
```

Comment:
Split this SQL into Roles Table and OpportunityReportView And move them to SSDT Project